types/tailwind-shades/tailwind-shades-tests.ts

```diff
@@ -0,0 +1,5 @@
+import shadesOf from "tailwind-shades";
```

Comment:
Use the commonjs syntax `import shadesOf = require('shadesOf')`. This syntax should only be used for default imports of ES modules, but tailwind-shades is uses commonjs `module.exports = shadesOf` in the original source.

Consumers of the package who want to use `import shadesOf from 'shadesOf'` will need to add `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.